packages/runtime-core/__tests__/apiOptions.spec.ts

```diff
@@ -539,7 +539,7 @@ describe('api: options', () => {
       expect('Invalid watch option: "foo"').toHaveBeenWarned()
     })
 
-    test('computed warn with setter and no getter', () => {
+    test('computed with setter and no getter', () => {
```

Comment:
I think the previous description is a bit better describing what the test is

Reply:
> I think the previous description is a bit better describing what the test is

but it wrapped in warnings describe, so i have no idea which is better.